scm/scm_invite.go

```diff
@@ -0,0 +1,28 @@
+package scm
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+)
+
+type SCMInvite interface {
+	// Accepts repository invites.
+	AcceptRepositoryInvites(context.Context, *RepositoryInvitationOptions) error
+}
+
+// RepositoryInvitationOptions contains information on which organization and user to accept invitations for.
+type RepositoryInvitationOptions struct {
+	Login string // GitHub username.
+	Owner string // Name of the organization.
+	Scm   SCM
```

Comment:
Note to self: `Scm` should be renamed to `ScmApp` to signal that it must be an SCM app implementation.